src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -616,31 +623,33 @@ private void testShutdown( boolean shouldThrow ) throws ClassNotFoundException,
             assertTrue( testDaemon.hasRun( ) );
             assertTrue( testDaemon2.hasRun( ) );
             Thread.sleep( 10L ); // leave some time to the daemons to exit
-                                 // executing daemons
+            // executing daemons
+
             scheduler.shutdown( );
             assertEquals( 1, testDaemon.getStopCallNumber( ) );
             assertEquals( 1, testDaemon2.getStopCallNumber( ) );
         }
+        catch( InterruptedException | BrokenBarrierException | TimeoutException e )
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?